azure/services/publicips/publicips.go

```diff
@@ -75,6 +75,9 @@ func (s *Service) Reconcile(ctx context.Context) error {
 	ctx, _, done := tele.StartSpanWithLogger(ctx, "publicips.Service.Reconcile")
 	defer done()
 
+	ctx, cancel := context.WithTimeout(ctx, s.Scope.DefaultedAzureServiceReconcileTimeout())
```

Comment:
Publicips didn't have an explicit timeout before this? 👍🏻

Reply:
correct, it must have been forgotten in the async refactor